Closing-the-object-via-event-target/test.js

```diff
@@ -0,0 +1,21 @@
+const divCollectionPane = document.querySelectorAll(".pane");
+const div = document.querySelector(".pane");
```

Comment:
you already have a `divCollectionPane`, get the one you need from `divCollectionPane`.
It is better to avoid unnecessary DOM searches